client/src/controllers/KeyboardController.ts

```diff
@@ -57,7 +64,14 @@ export class KeyboardController extends Controller<
   declare scopeValue: '' | 'global';
 
   /** Allow an explicit target to be the element that the keyboard activates instead of the controlled element. */
-  declare readonly elementTargets: (HTMLButtonElement | HTMLAnchorElement)[];
+  declare readonly elementTargets: (
+    | HTMLButtonElement
+    | HTMLAnchorElement
+    | HTMLInputElement
+  )[];
```

Comment:
We no longer need this change, let's just make it simple and use `HTMLElement`. I can do this when merging.